### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by serentinos**
[DEMO LINK](https://serentinos.github.io/react_keyboard/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  key: string;
+}
+
+export class App extends Component<{}, State> {
+  state = {
+    key: '',
```
Comment (33Orange) [inline src/App.tsx:9]: Use `null` instead empty string for initial empty value

File: src/App.tsx [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.keyboardHandler);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.keyboardHandler);
+  }
+
+  keyboardHandler = (event: KeyboardEvent) => {
```
Comment (33Orange) [inline src/App.tsx:20]:
```suggestion
  handleKeyUp= (event: KeyboardEvent) => {
```

**Review by 33Orange — CHANGES_REQUESTED** [review]
Almost done

A few moments to fix

**Review by naziamoff — APPROVED** [review]
awesome 🔥

**Pull request closed without merging**